CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * `pro/generate-documentation-qmd`: Added `working_directory` option. 
 
+## Bug fixes
+
+* `project/detect-changed-components`: Use `path` and `parent` to detect a component's resources.
```

Comment:
Could you add that `path:` and `parent:` originate from the viash config? I first interpreted this as new arguments being added.